mlem/contrib/docker/install_req.j2

```diff
@@ -1,4 +1,5 @@
 {% if packages %}RUN {{ package_install_cmd }} {{ packages|join(" ")  }}{% endif %}
+RUN pip install --upgrade pip
```

Comment:
just wondering if this really neccessary

Reply:
removed